src/blocks/donate/frontend/class-newspack-blocks-donate-renderer-base.php

```diff
@@ -33,8 +50,10 @@ public static function get_configuration( $attributes ) {
 
 		$configuration['defaultFrequency'] = $attributes['defaultFrequency'];
 
+		$is_manual = Newspack_Blocks::can_use_name_your_price() && $attributes['manual'];
+
 		/* If block is in "manual" mode, override certain state properties with values stored in attributes */
-		if ( $attributes['manual'] ?? false ) {
+		if ( $is_manual ?? false ) {
```

Comment:
```suggestion
		if ( $is_manual ) {
```

The `?? false` is superfluous if we're not checking if `manual` is set in the associative array.